README.md

```diff
@@ -16,6 +16,17 @@ The result of the sync is logged to a sync log available from the settings scree
 
 Only simple products are synced. Configurable products are not synced.
 
+## Requirements
+
+In order for the cron to successfully process the listings sync in parallel-thread manner, the magento cron needs to be
+declared in the crontab via the following:
+
+php cron.php -mdefault 1
```

Comment:
But...this is not cron syntax? Am I missing something? Shouldn't every line start with timing information? "* * * * *"